### PR TITLE
[IOTDB-5284] Fix some code smells

### PR DESCRIPTION
**Pull request by BUAAserein**
### Fix some code smells in confignode[iotdb-confignode], please see in [JIRA5284](https://issues.apache.org/jira/browse/IOTDB-5284)

### CODE REVIEW

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/cq/CQScheduleTask.java [hunk 1]
```diff
@@ -216,6 +204,18 @@ public AsyncExecuteCQCallback(long startTime, long endTime) {
...
+        executionTime = executionTime + everyInterval;
+      } else if (timeoutPolicy == TimeoutPolicy.DISCARD) {
+        long now = System.currentTimeMillis();
+        executionTime =
+            executionTime + ((now - executionTime - 1) / everyInterval + 1) * everyInterval;
+      } else {
+        throw new IllegalArgumentException("Unknown TimeoutPolicy: " + timeoutPolicy);
+      }
+    }
+
```
Comment (SpriCoder) [inline confignode/src/main/java/org/apache/iotdb/confignode/manager/cq/CQScheduleTask.java:218]: Why move these method?

**Review by SpriCoder — CHANGES_REQUESTED** [review]
PTAL~

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/cq/CQScheduleTask.java (hunk 1 above)
Comment (BUAAserein) [reply confignode/src/main/java/org/apache/iotdb/confignode/manager/cq/CQScheduleTask.java:218]: "private" methods called only by inner classes should be moved to those classes

Comment (SpriCoder) [reply confignode/src/main/java/org/apache/iotdb/confignode/manager/cq/CQScheduleTask.java:218]: Get it!

**Review by SpriCoder — APPROVED** [review]
LGTM

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Pull request merged**